### ISSUE TITLE
[enhancement] Automatized build, DMG and Notarization

### ISSUE THREAD

**Issue by EETagent** [MSzturc/ThinkpadAssistant#41]
Continuation of the discussion from #40 

TLDR: It would be great to have fully automatized build procedure with .app and .dmg sign, notarization and release.

What we have so far:

- [x] Build .app
- [x] Sign .app
- [x] Package .dmg
- [x] [Release](https://github.com/MSzturc/ThinkpadAssistant/releases/tag/v1.9.2.1) .dmg
- [x] Get commit messages since last tag and post them as a release description ( not merged currently )

Paid certificate is needed so this won't be implemented:
- [ ] Sign .dmg
- [ ] Notarize .app
- [ ] Notarize .dmg

Possibly in the future:

 [?] Generate AppCast.xml #26 

Currently untested [release.yml](https://github.com/EETagent/ThinkpadAssistant/blob/master/.github/workflows/release.yml)

Notarization is [requirement](https://developer.apple.com/news/?id=10032019a) for secure app distribution outside of ( and in ) an App Store. Without it ( Since Catalina ), user would see unknown developer warning and would have to manually allow first app launch through preferences ( Does not affect machines where is Gatekeeper disabled `sudo spctl --master-disable` ).
![obrazek](https://user-images.githubusercontent.com/20557318/89221252-40c9bc80-d5d3-11ea-9cc4-5eaa188adba0.png)

**Comment by MSzturc** [comment]
That's bad. Im not able to reproduce this error. I guess since since my certificate is in my keychain it detects that it's my own software and starts without this error..

**Comment by MSzturc** [comment]
Aww.. i've noticed that a paid account is needed for the notarize program, my free developer account is not enough. This account costs 99$ a year.

**Comment by EETagent** [comment]
Oh, I thought you have a paid cert. We can bypass that then.

**Comment by EETagent** [comment]
@MSzturc 

I removed all notarization things. There is still .app signature check though
